proto/api/users.proto

```diff
@@ -1,10 +1,13 @@
 syntax = "proto3";
 
 package api;
+
+option go_package = "gohire/proto/gen/api";
   
 service UserService {
     // rpc SignIn(SignInRequest) returns (SignInResponse) {}
     rpc GetUsers(GetUsersRequest) returns (GetUsersResponse) {}
+    rpc PostUsers(PostUsersRequest) returns (PostUsersResponse) {}
```

Comment:
How did you decide on "PostUsers" as the name for a server API?

Reply:
I was thinking in terms of building out CRUD endpoints GET /users POST /users etc as well as sticking to the convention that the app seemed to be going for with GetUsers. Perhaps using "Create" in the name rather than "Post" here would be better/preferred?